clients/src/main/java/org/apache/kafka/clients/NodeApiVersions.java

```diff
@@ -37,18 +36,18 @@
 public class NodeApiVersions {
 
     // A map of the usable versions of each API, keyed by the ApiKeys instance
-    private final Map<ApiKeys, ApiVersion> supportedVersions = new EnumMap<>(ApiKeys.class);
+    private final Map<ApiKeys, ApiVersionsResponseKey> supportedVersions = new EnumMap<>(ApiKeys.class);
```

Comment:
Should we rename the generated class to have a more concise name? It seems unnecessarily verbose.

Reply:
How about renaming ```ApiVersionsResponseKey``` to ```ApiVersion```?

Reply:
Using `ApiVersion` sounds reasonable to me.

Reply:
Sounds good if there are no name conflicts as a result of that change.

Reply:
I have renamed it in the latest commit.